static/docker-compose.yml

```diff
@@ -5,41 +5,40 @@ services:
       update_config:
         parallelism: 1
     depends_on:
-    - plextracdb
-    - redis
-    - postgres
+      - plextracdb
+      - redis
+      - postgres
     environment:
       STARTUP_MODE: API_ONLY
       CLOUD_STORAGE_ENDPOINT: ${CLOUD_STORAGE_ENDPOINT:-minio}
       CLOUD_STORAGE_SSL: ${CLOUD_STORAGE_SSL:-false}
     env_file:
-    - .env
+      - .env
```

Comment:
the core-backend will get the new env vars via this.  Not ideal, but it is what it is.